_data/changelogs/component-form.yml

```
@@ -5,6 +5,14 @@ title: Form
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Fixed a bug that caused standard text input variants to show disabled styles when in forced colors mode.
+    summaryAdditional: Now the disabled border color only shows when the element is disabled.
+    affectsStyles: true
+    affectsAccessibility: true
+    githubPr: 5397
+    githubRepo: uswds
+    versionUswds: 3.5.1
```

```suggestion
    versionUswds: N.N.N
```

While this is almost certainly going to be 3.5.1, I used dummy text here for consistency